app/views/liquids/_codepen.html.erb

```diff
@@ -0,0 +1,8 @@
+<iframe
+  height="<%= height %>"
+  src="<%= "#{link}?height=#{height}&#{build_options}&embed-version=2" %>"
```

Comment:
Below is the correct way to use the ERB tag. If the whole URI is wrapped in ERB tag, the & and # would be escaped.
```suggestion
  src="<%= link %>?height=<%= height %>&#<%= build_options %>&embed-version=2"
```